<launch>
    <node name="cameraCalib" pkg="abb2400_sim" type="cameraCalib" output="screen">
        <param name="cam_type" value="astra" />
        <remap from="DepthImage" to="/camera/depth/image_raw"/>
        <remap from="ColorImage" to="/camera/rgb/image_raw"/>
    </node>
</launch>